syntax = "proto3";

package water_pump_service;

service WaterPumpService {
  rpc TurnOnWaterPump (TurnOnWaterPumpRequest) returns (Status);
  rpc TurnOffWaterPump (TurnOffWaterPumpRequest) returns (Status);
}

message TurnOnWaterPumpRequest {
  // Campos do pedido para ligar a bomda de 치gua, se necess치rio
}

message TurnOffWaterPumpRequest {
  // Campos do pedido para desligar a bomda de 치gua, se necess치rio
}

message Status {
  string message = 1;
}
